empyrical/stats.py

```diff
@@ -152,6 +152,34 @@ def cum_returns(returns, starting_value=0):
         return df_cum * starting_value
 
 
+def total_returns(returns, starting_value=0):
+    """
+    Compute total returns from simple returns.
+
+    Parameters
+    ----------
+    returns : pd.Series or np.ndarray
+        Returns of the strategy as a percentage, noncumulative.
+         - Time series with decimal returns.
+         - Example:
+            2015-07-16    -0.012143
+            2015-07-17    0.045350
+            2015-07-20    0.030957
+            2015-07-21    0.004902.
+    starting_value : float, optional
+       The starting returns.
+
+    Returns
+    -------
+    float
+
+
+
+    """
+
+    return cum_returns(returns, starting_value).iloc[-1]
```

Comment:
If we pass an ndarray, then there's no `iloc` attribute, right?  

Want to add a test to the suite?  If you add it to the base class, the subclasses will try with different input types.